Sources/ATTNAPI.m

```diff
@@ -189,6 +192,38 @@ - (void)sendEventInternalForRequest:(EventRequest*)request userIdentity: (ATTNUs
         orderConfirmedMetadata[@"products"] = [self convertObjectToJson:products defaultValue:@"[]"];
         [eventRequests addObject:[[EventRequest alloc] initWithMetadata:orderConfirmedMetadata eventNameAbbreviation:@"oc"]];
         
+        return eventRequests;
+    } else if ([event isKindOfClass:[ATTNAddToCartEvent class]]) {
+        ATTNAddToCartEvent* addToCart = (ATTNAddToCartEvent*)event;
+        
+        if ([addToCart.items count] == 0) {
+            NSLog(@"No items found in the AddToCart event.");
+            return @[];
+        }
+        
+        for (ATTNItem* item in addToCart.items) {
+            NSMutableDictionary* metadata = [[NSMutableDictionary alloc] init];
+            [self addProductDataToDictionary:item dictionary:metadata];
+            
+            [eventRequests addObject:[[EventRequest alloc] initWithMetadata:metadata eventNameAbbreviation:@"c"]];
```

Comment:
nit: should we make these abbreviations constants like we do for the user identity keys?

Reply:
good call, literal strings are lazy. updated!